tests/test_flows_view.py

```diff
@@ -0,0 +1,75 @@
+from mock import patch
+
+# Must import test base before the Controller
+from base import FlowTest, FlowStep
+
+from seedsigner.controller import StopFlowBasedTest
+from seedsigner.gui.screens.screen import RET_CODE__POWER_BUTTON
+from seedsigner.models.settings import Settings
+from seedsigner.views.tools_views import ToolsCalcFinalWordNumWordsView, ToolsMenuView
+from seedsigner.views.view import MainMenuView, NotYetImplementedView, PowerOptionsView, PowerOffView, RestartView, UnhandledExceptionView, View
+
+
+
+class TestViewFlows(FlowTest):
+
+    def test_restart_flow(self):
+        """
+        Basic flow from MainMenuView to RestartView
+        """
+        with patch('seedsigner.views.view.RestartView.DoResetThread'):
+            self.run_sequence([
+                FlowStep(MainMenuView, screen_return_value=RET_CODE__POWER_BUTTON),
+                FlowStep(PowerOptionsView, button_data_selection=PowerOptionsView.RESET),
+                FlowStep(RestartView, screen_return_value=StopFlowBasedTest()),
```

Comment:
I suspect that `screen_return_value=StopFlowBasedTest()` here and in the test methods below are a left-over from flow tests before they evolved not to need it.  This small test module is so clean, and because of it's name, it's likely to be used for learning by future devs, leading them to re-introduce this no-longer-needed end-of-flow-test pattern.  If not needed, also might not need to import that exception.